ProfessorAssessor/Models/Professor.m

```diff
@@ -11,4 +14,20 @@ + (nonnull NSString *)parseClassName {
     return @"Professor";
 }
 
++ (Professor *)professorFromPFObject:(PFObject *)object {
+    Professor *professor = [Professor new];
+
+    if (object) {
+        professor.identifier = object[@"objectId"];
```

Comment:
How do you know that PFObject will have any of this information? It can be anything. What happens if you accidentally pass it the PFObject for a course?